cookbooks/index.md

```diff
@@ -0,0 +1,11 @@
+(part:user_manual:chap:cookbooks:sec:index)=
+Cookbooks
+======================
+
+This section contains self contained cookbooks on how to design different geodynamic setups. Folders contain World Builder parameter files, grid output files, and optionally markdown files that are incorporated into the online documentation and explain particular features about the setup.
```

Comment:
```suggestion
This section contains self-contained cookbooks on how to design different geodynamic setups. Folders contain World Builder parameter files, grid output files, and optionally a doc folder that contains relevant figures and markdown files that are incorporated into the online documentation and explain particular features about the setup.
```

Or if you have a good idea for how not to repeat 'contained' that would be even better.